contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -176,6 +177,10 @@
       },
     },
     mounted() {
+      this.resetStaticMathFieldsObserver = new ResizeObserver(() => {
```

Comment:
todo: call `this.resetStaticMathFieldsObserver.disconnect()` when cleaning up the `MarkdownEditor` component